src/test/java/stepdefs/CucumberHook.java

```diff
@@ -0,0 +1,50 @@
+package stepdefs;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.Selenide;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.OutputType;
+
+import java.util.Base64;
+
+@Slf4j
+public class CucumberHook {
+
+  @Before
+  public void beforeScenario(Scenario scenario) {
```

Comment:
what is the reason to pass configurations before each scenario?

Reply:
yes, you right, thanks, removed and moved to selenide property.